controlpanel/frontend/jinja2/datasource-list.html

```diff
@@ -19,6 +19,18 @@
 {% block content %}
 <h1 class="govuk-heading-xl">{{ page_title }}</h1>
 
+{% if all_datasources and request.user.has_perm('api.is_superuser') %}
```

Comment:
Is this intended as a one-time action? And then will we remove it afterwards? If thats the case thats fine, in retrospect could have added a [data migration](https://docs.djangoproject.com/en/5.1/topics/migrations/#data-migrations) to loop through the buckets and do the update, to save needing to implement a temporary view etc